tooling/create-helium-app/template-base-essentials

```diff
@@ -0,0 +1 @@
+/Users/madison.kneller/ThoughtIndustries/helium-folder/helium/tooling/template-base-essentials
```

Comment:
not sure about this ?

Reply:
@rjschill87 Updated the symlink to use a relative path for `template-base-essentials`, but for `template-base`, I'm a bit confused, is what you linked not the correct symlink?

Reply:
@madison-kneller-ti You're correct! I apologize, misread the diff as replacing the existing symlink vs. creating a new one.